distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -163,9 +163,22 @@ type Primitive
        Returns the (primitive) value wrapped by this type
     value self = self.impl
 
+
+type Function
+    private Value impl kind:Integer
+
+    is_resolved self -> Boolean =
```

Comment:
Add docs. What exactly does `is_resolved` mean in this context? Is it "resolved to a concrete method/constructor at runtime / compile time?"

Reply:
Docs added in 0392281